backend/app/controllers/api/v1/accounts/project_developers_controller.rb

```diff
@@ -16,11 +15,23 @@ def create
           end
         end
 
+        def update
+          if current_user.role.to_sym != :project_developer
+            raise API::UnprocessableEntityError, I18n.t("errors.messages.user.no_project_developer")
+          end
+
+          current_user.account.update! account_params.except(:language)
+          current_user.account.project_developer.update! project_developer_params.except(:language)
+          render json: ProjectDeveloperSerializer.new(current_user.account.project_developer).serializable_hash
+        end
+
         private
 
         def account_params
-          params.fetch(:project_developer_params, params)
+          data = params.fetch(:project_developer_params, params)
             .permit :language, :picture, :name, :website, :linkedin, :facebook, :twitter, :instagram, :about
+          data = data.except(:picture) if data[:picture].blank?
```

Comment:
This is here to just prevent `ActiveSupport::MessageVerifier::InvalidSignature` in case that you send empty string via rswag

Reply:
Hmm, maybe a comment to remove it if we remove multipart form? Normally if no one wants to update picture it won't be present in body.

Reply:
If we will remember to checkout `Send empty values` for `picture` all the time when using rswag, I can remove this, np

Reply:
Yeah, then maybe remove it, as this is not code related, but more rswag and api docs related.